cmd/notary/keys.go

```diff
@@ -258,24 +267,43 @@ func (k *keyCommander) keysExportRoot(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
-	cs := cryptoservice.NewCryptoService("", ks...)
+
+	// Search for this key in all of our keystores, determine whether this key has a GUN
```

Comment:
We care about the GUN so we can create a cryptoservice? The flatkeystore will help with this, correct?